solution/ui/prototype/src/components/resources/ResourcesResults.vue

```diff
@@ -122,31 +138,35 @@ export default {
                                 item.sub_category = category.name;
                             } else {
                                 item.category = category.name;
-                            }                            
+                            }
                             returnArr.push(item);
                         });
                     }
 
                     return returnArr;
                 });
+
             //remove duplicates
-            x = _uniqBy(x, (item) => {
+            results = _uniqBy(results, (item) => {
                 return item.name;
             });
-            return x;
+
+            return results;
         },
     },
 
-    methods: {
-        methodName() {
-            console.log("method has been invoked");
+    methods: {},
+
+    filters: {
+        locationLabel(value) {
+            return value.substring(3);
         },
     },
 
     watch: {
```

Comment:
do we even need the watch block at all?

Reply:
Good catch - I went in and removed a bunch of unused methods.